Common/brushed_dc.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+namespace sjsu
+{
+    class brushed_dc
+    {
+    public:
+        /// Initialize peripherals required to communicate with device. Must be the
+        /// first method called on this driver. After this is called, `Enable()` can
+        /// be called.
+        virtual bool Initilize() = 0;
+        /// This method must be called before running `Read()`. This function will
+        /// configure the device settings as defined through the constructor of this
+        /// interface's implementation. Some implementations have more detail or
+        /// settings than others.
+        virtual bool Enable() = 0;
+        virtual bool DriveForward() = 0;
```

Comment:
Add comments to each method in this class. The comment must dictate the exact behavior expected by the interface.